test/spectests/spectests.cpp

```diff
@@ -88,6 +88,23 @@ class test_runner
                 }
                 pass();
             }
+            else if (type == "register")
+            {
+                const auto module_name =
+                    (cmd.find("name") != cmd.end() ? cmd["name"] : UnnamedModule);
+
+                const auto it_instance = instances.find(module_name);
+                if (it_instance == instances.end())
+                {
+                    skip("Module not found.");
```

Comment:
Is this a skip or a fail?

Reply:
I think skip is better, because typically it would mean than another module failed to instantiate, and then any following tests using it are skipped.